### DISCUSSION TITLE
Get formState on child input component

### DISCUSSION THREAD

**Discussion by viniciusmurad** [react-hook-form/react-hook-form#3441] [General] [answered]
I'm trying to send the formstate to my child input component,but when I pass the formstate props to my component, i got undefined.

I put my code in the [codepan](https://codesandbox.io/s/react-hook-form-smart-form-component-forked-dfxuy?file=/src/components.js:139-145)

Is there another way to access the dirtyFields property?

**Comment by bluebill1049** [comment, marked as answer]
UseFormContext?
